modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/emodelrelease/v0.1.2.json

```diff
@@ -0,0 +1,38 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/emodelrelease/v0.1.2/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/simulation/modelreleaseindex/v0.1.1"
```

Comment:
It should be modelrelease instead of modelreleaseindex

Reply:
Below a modelrelease/v0.1.2 is used and not modelrelease/v0.1.1. I would suggest to use v everywhere.

Reply:
done

Reply:
I'm not seeing any change.